aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -1554,6 +1554,20 @@ Tensor _cholesky_solve_helper_cuda(const Tensor& self, const Tensor& A, bool upp
   return self_working_copy;
 }
 
+// Todo: cusolverDn<T>potrsBatched only supports nrhs == 1 and does not have good performance.
```

Comment:
Should we link this comment to the cuSOLVER issue?

Reply:
This is noted in the cusolver doc, so it's more like a "feature". I'll fill a feature request to add support for nrhs > 1 cases.